c7n/resources/efs.py

```diff
@@ -193,3 +194,42 @@ def process(self, resources):
                     LifecyclePolicies=op_map.get(self.data.get('state')))
             except client.exceptions.FileSystemNotFound:
                 continue
+
+
+@ElasticFileSystem.filter_registry.register('lifecycle-policy-enabled')
+class LifecyclePolicyEnabled(Filter):
+    """Filters efs based on the state of lifecycle policies
+
+    :example:
+
+      .. code-block:: yaml
+
+            policies:
+              - name: efs-filter-lifecycle
+                resource: efs
+                filters:
+                  - type: lifecycle-policy-enabled
+                    state: true
+
+    """
+    schema = type_schema(
+        'lifecycle-policy-enabled',
```

Comment:
perhaps drop the enabled suffix since we're doing the state boolean

Reply:
Sounds good!